be/src/olap/rowset/segment_v2/binary_dict_page.cpp

```diff
@@ -173,6 +189,7 @@ Status BinaryDictPageDecoder::init() {
     }
 
     RETURN_IF_ERROR(_data_page_decoder->init());
+    RETURN_IF_ERROR(_dict_decoder->init());
```

Comment:
if encoding_type is plain, _dict_decoder is nullptr. That will lead to crash

Reply:
👌